themes/default/layouts/partials/hand-raise-section.html

```diff
@@ -42,13 +42,13 @@ <h5>Need help getting started?</h5>
                     {{ partial "color-icon.html" (dict "icon" "team" "icon_color" "blue") }}
                 </div>
                 <div>
-                    <h5>Want to speak with sales?</h5>
+                    <h5>Want a demo?</h5>
```

Comment:
Can you walk me through all the /layout/partials changes. I am not too sure where to look for the change on the preview link